docs/user-guide/faq.md

```diff
@@ -115,3 +115,27 @@ Refer to [this](about-rules.md#-empty-line-before-and--max-empty-lines) section
 ## How do I "pin" to a particular branch, pull request or fork of stylelint in my `package.json`?
 
 Use [`npm-git-install`](https://github.com/lzrski/npm-git-install) to work around [a current limitation](https://github.com/npm/npm/issues/3055) of `npm`.
+
+## If I extend a configuration, will the options for each rule be merged or overridden?
+
+In most cases rules will be merged but **secondary options will be overriden**.
```

Comment:
When I used "merged", I was thinking about this scenario

`config-1.json`

```json
{
 "rules": {
   "font-weight-notation": "numeric",
   "number-leading-zero": "always",
   "at-rule-no-unknown": [true, {
     "ignoreAtRules": [
       "at-root",
       "value"
     ]
   }]
 }
}
```

`config-2.json`

```json
{
 "extends": "./config-1.json",
 "rules": {
   "font-weight-notation": "named-where-possible",
   "at-rule-no-unknown": [true, {
     "ignoreAtRules": [
       "extends"
     ]
   }]
 }
}
```

Result

```json
{
 "extends": "./config-1.json",
 "rules": {
   "font-weight-notation": "named-where-possible", // this is overriden
   "number-leading-zero": "always", // this is merged from config-1
   "at-rule-no-unknown": [true, {
     "ignoreAtRules": [ // this is overriden
       "extends"
     ]
   }]
 }
}
```